content/geometry/general.adoc

```diff
@@ -83,11 +83,43 @@ NOTE: Origins of the coordinate frames for specific object classes are under
 discussion.
 
 == Naming conventions
+=== General
 The following naming conventions apply to ASAM OpenMATERIAL geometry files:
 
-* The name of a 3D model file shall have the prefix `omg_` to indicate that the file
-complies with the ASAM OpenMATERIAL geometry specification.
-* The 3D model file and the related 3D asset file shall have the same base name.
+* The name of a 3D model file shall have the prefix "omg_" to indicate that the file complies with the ASAM OpenMATERIAL geometry specification.
+* The 3D model file and the related 3D asset file must have the same base name.
+  Example: omg_Example.gltf, omg_Example.xoma
+* The naming convention inside the 3D model file (contains 3D informations) must follow the snake_case definition, to better human readability and consistent parsing of the file structure
+* The naming convention inside the 3D asset file (contains meta data) must follow the lowerCamelCase definition, to allow a consistent naming convention in all .json files and an consistent parsing.
+* LOD levels (Level of Detail) are not part of the first version of the standard, but will be added in future versions for each asset type.
+* Subkeys are predefined and not (yet) standardized Subkeys are recommended to use.
```

Comment:
I'm not familiar with the term subkeys. From the context I understand it's a keyword. Would this be more suitable?